yarn.lock

```diff
@@ -15800,6 +15775,13 @@ ovh-api-services@^11.0.0, ovh-api-services@^11.0.1:
   dependencies:
     lodash "^4.17.20"
 
+ovh-api-services@^11.1.0:
```

Comment:
Could you please rebase on `develop` branch (and re-run `yarn`).

Both `package.json` and `yarn.lock` shouldn't be part of the diff here (since it has been already bumped)